pkgs/applications/networking/cluster/kompose/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "kompose-${version}";
+  version = "v1.5.0";
```

Comment:
The common pattern in nixpkgs is to have the version number be just `version = "1.5.0";` and the rev in `fetchFromGithub` be `rev = "v${version}";`